code/modules/surgery/bodyparts/head.dm

```diff
@@ -0,0 +1,163 @@
+
+
+/obj/item/bodypart/head
+	name = "head"
+	desc = "what a way to get a head in life..."
```

Comment:
"Didn't make sense not to live for fun, your brain gets smart but your head gets dumb."